lib/travis/api/v3/queries/log.rb

```diff
@@ -8,7 +8,7 @@ def find(job)
       raise EntityMissing, 'log not found'.freeze if log.nil?
       #if the log has been archived, go to s3
       if log.archived_at
-        content = fetch.get(prefix).try(:body)
+        content = fetch.get(prefix).try(:body).force_encoding('UTF-8')
```

Comment:
OK, that might be valid UTF-8, not sure, would be interesting to know if logs might be in a different encoding at this point. IIRC we clean up bytes at one point, if that's still on the path (using https://github.com/rkh/coder).

If the string contains bytes that aren't valid UTF-8 and you force set the encoding to UTF-8, unexpected errors might pop up down the line.

Also, `try(:body)` might return `nil`, which doesn't implement `force_encoding`, resulting in an exception.